includes/abstracts/abstract-wc-product.php

```diff
@@ -1420,6 +1420,13 @@ public function is_visible() {
 			$visible = false;
 		}
 
+		$parent_id = $this->get_parent_id();
+		if ( ! empty( $parent_id ) ) {
```

Comment:
You don't need empty here.

https://stackoverflow.com/questions/1075534/cant-use-method-return-value-in-write-context#answer-4328049

Reply:
Thanks for the info. Removed the empty from the if statement and now just testing on the returned value.